astrobee_api/scripts/genBaseRobot.py

```diff
@@ -11,9 +11,9 @@
 import logging
 
 # hack to ensure xgds_planner2 submodule is at head of PYTHONPATH
-ffroot = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
-sys.path.insert(0, os.path.join(ffroot, 'common', 'plans', 'xgds_planner2'))
-sys.path.insert(0, os.path.join(ffroot, 'common', 'scripts'))
+astrobee_root = os.getenv('SOURCE_PATH', (os.environ['HOME'] + "/astrobee"))
```

Comment:
Does this mean that the SOURCE_PATH needs to be defined at all times if we don't have astrobee in the HOME folder? Might be better to use relative path as we know astrobee_android is in the submodules folder of astrobee_root?